core/backend/src/test/standalone/IModel.test.ts

```diff
@@ -1688,6 +1695,214 @@ describe("iModel", () => {
     iModel.close();
   });
 
+  it("should be able to create a snapshot IModel and set geolocation by GCS", async () => {
+    const args = {
+      rootSubject: { name: "TestSubject", description: "test project" },
+      client: "ABC Engineering",
+      globalOrigin: { x: 10, y: 10 },
+      projectExtents: { low: { x: -300, y: -300, z: -20 }, high: { x: 500, y: 500, z: 400 } },
+      guid: Guid.createValue(),
+    };
+
+    const gcs: GeographicCRS = new GeographicCRS({

```

Comment:
just a nitpick, but there's no reason to declare the type twice.

Reply:
Done.